migration-tool/pkg/log/log.go

```diff
@@ -0,0 +1,100 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license
+
+// Package log creates logs in the same way as Prometheus, while ignoring errors
+package log
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+)
+
+var (
```

Comment:
Can you please add a comment that this is a copy/paste of the same file from Promscale and that hopefully in the future we find some better way to handle this.